frontend/src/app/forms/forms.page.ts

```diff
@@ -164,6 +164,11 @@ export class FormsPage
    */
   private hasSubmittedForm = false;
 
+  /**
+   * If a skip is stepped, skipAlertShowed = true, if no stepp is skipped till now, it returns false.
```

Comment:
Uhm, I think your comment message is a bit broken :D 

From my point of view, you could maybe summarize it like so:
"Has the user ever skipped a step?"

Reply:
Changed it ;)